StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -1175,3 +1183,37 @@ bool StPicoDstMaker::selectVertex()
   // Retrun false if selected vertex is not valid
   return selectedVertex ? true : false;
 }
+
+void StPicoDstMaker::fillFmsHits()
+{
+  /*
+  StFmsDbMaker* fmsDb=static_cast<StFmsDbMaker*>(GetMaker("fmsDb"));//can save this into a member variable
```

Comment:
Yes, this would be a better place to access StFmsDbMaker or St_db_Maker if you need to update the FMS hit data before storing them